sdk/azidentity/environment_credential_test.go

```diff
@@ -278,6 +283,7 @@ func TestEnvironmentCredential_InvalidClientSecretLive(t *testing.T) {
 
 func TestEnvironmentCredential_UserPasswordLive(t *testing.T) {
 	vars := map[string]string{
+		//for ADFS change clientID
 		azureClientID: developerSignOnClientID,
```

Comment:
This developerSignOnClientID doesn't work with ADFS. For testing, I had to change this client ID to be someting that ADFS accepts, such as Windows Powershell.